content/documentation/pages/6-recipes/7-scaling/2-autoscaling.md

```diff
@@ -0,0 +1,180 @@
+The `spring.cloud.dataflow.client.server-uri` property is used to configure the Data Flow Scale API endpoint.
+For the AlertWebHook configurations check [alertwebhook-deployment.yaml]().
+
+The `Data Flow Scale REST API` provides a platform agnostic mechanism for scaling pipeline applications and exposes the following REST interface:
+
+```shell
+POST /streams/deployments/scale/{streamName}/{appName}/instances/{count}
+```
+
+- `streamName` - stream pipeline name.
+- `appName` - application name (in the stream pipeline) to scale.
+- `count` - desired number of application instances.
+
+## Prerequisite
+
+This recipe uses the Kubernetes platform and Kafka middleware.
+
+<!--TABS-->
+
+<!--For Kubectl installation-->
+
+- Follow the [Kubectl](%currentPath%/installation/kubernetes/kubectl/) installation instructions to setup Spring Cloud Data Flow with Kafka broker.
+
+- Then install the `Alertmanager` and the `AlertWebHook` and reconfigure the `Prometheus` services:
+
+  ```shell
+  kubectl create -f ./kubernetes/alertwebhook/
+  kubectl create -f ./kubernetes/alertmanager/
+  kubectl patch cm prometheus --patch "$(cat ./kubernetes/prometheus/prometheus-configmap.yaml)"
+  kubectl delete pods -l app=prometheus
+  ```
+
+<!--For Helm installation-->
+
+- Follow the [Helm](%currentPath%/installation/kubernetes/helm/) installation instructions to setup Spring Cloud Data Flow with Kafka broker. You can use `features.monitoring.enabled=true` with at least `10Gi` storage space:
+
+  ```shell
+  helm install --name my-release stable/spring-cloud-data-flow --set features.monitoring.enabled=true,kafka.enabled=true,rabbitmq.enabled=false,kafka.persistence.size=10Gi
+  ```
+
+- Then install the `Alertmanager` and the `AlertWebHook` and reconfigure the `Prometheus` services:
+
+  ```shell
+  kubectl create -f ./kubernetes/helm/alertwebhook/
+  kubectl create -f ./kubernetes/alertmanager/
+  kubectl patch cm my-release-prometheus-server --patch "$(cat ./kubernetes/helm/prometheus/prometheus-configmap.yaml)"
+  ```
+
+<!--END_TABS-->
+
+Install latest [kafka-docker](https://dataflow.spring.io/kafka-docker-latest) app starters:
+
+```shell
+app import --uri https://dataflow.spring.io/kafka-docker-latest --force
+```
+
+## Autoscaling Recipe
+
+### Create stream pipeline
+
+```shell
+stream create --name scaletest --definition "time --fixed-delay=995 --time-unit=MILLISECONDS | transform --expression=\"payload + '-' + T(java.lang.Math).exp(700)\" | log"
+```
+
+The `time` source generates timestamp messages on fixed time intervals (~ 1 msg/s), the `transform` processor performs a math operation to simulates a CPU intensive processing and the `log` sink prints the message payload.
```

Comment:
PLEASE use the same text that I put in change suggestion. Just copy-paste it here.